### PR TITLE
Separate meca path with reviews

### PR DESCRIPTION
**Pull request by de-code**
part of https://github.com/elifesciences/data-hub-issues/issues/919
part of https://github.com/elifesciences/data-hub-issues/issues/914

This is the draft DocMap change that we discussed during the meeting.

Open questions:

- Is the meca path going to be the same
- What should the exact meca path be?

Note: Not meant to merge this PR as the file is used for regression tests (and need to be implemented first)

### CODE REVIEW

**Review by scottaubrey — APPROVED** [review]

**Review by fred-atherden — APPROVED** [review]

**Comment by de-code** [comment]
This now has become an output of `b1` (instead of `b2`). Would be good to re-review.

File: data/docmaps/regression_test/docmap_by_manuscript_id/80984.json [hunk 1]
```diff
@@ -253,6 +253,18 @@
                             ]
                         }
                     ]
+                }, {
+                    "participants": [],
+                    "outputs": [{
+                        "type": "computer-file",
```
Comment (scottaubrey) [inline data/docmaps/regression_test/docmap_by_manuscript_id/80984.json]: As per the [docmap framework documentation](https://docmaps.knowledgefutures.org/pub/sgkf1pqa#convention-summary), the output should be an [`expression`](https://sparontologies.github.io/fabio/current/fabio.html#d4e3239), but [`computer-file`](https://sparontologies.github.io/fabio/current/fabio.html#d4e2608) is a manifestation.

So output should recreate the values from the [draft RPP Thing](https://github.com/elifesciences/data-hub-api/pull/162/files#diff-71bfaaf7453740566a15d1f8cc5640f283ec98f62f187dbed739f1b197bcc341R24-R30), and attach `content` to that duplicated item.

**Review by scottaubrey — CHANGES_REQUESTED** [review]
Hey @de-code 

I've had a relook at this and I think I missed the first time that we have an incorrect typing. I've explained as a comment on the type line, but I think we should change the additional action:
```
{
  "type": "preprint",
  "identifier": "80984",
  "doi": "10.7554/eLife.80984.1",
  "versionIdentifier": "1",
  "content": [
    {
        "type": "computer-file",
        "url": "s3://prod-elife-epp-meca/reviewed-preprints/86284-v1-meca.zip"
    }
  ]
}
```
`license` and other properties are optional.

**Comment by de-code** [comment]
> I've had a relook at this and I think I missed the first time that we have an incorrect typing. I've explained as a comment on the type line, but I think we should change the additional action:

This has changed when it moved from `b2` to `b1`. I have updated it now as per your suggestion.

**Review by scottaubrey — APPROVED** [review]

**Comment by HazalCiplak** [comment]
This changes are deployed to the staging and example DocMaps with MECA path updated for regression test in [this related PR](https://github.com/elifesciences/data-hub-api/pull/184).
I am closing this PR. (No need to merge)

**Pull request closed without merging**